docs/pages/index.mdx

```diff
@@ -1,175 +1,220 @@
-Teleport.
+  <Admonition type="tip" title="Quick demo environments">
 
-You can also get started right away with a production-ready Teleport cluster.
-[Sign up for a free trial](https://goteleport.com/signup/) of Teleport
-Team.
+  For a quick demo environment you can use to follow this guide, consider
+  installing our DigitalOcean 1-Click droplet. View the installation page on
+  [DigitalOcean
+  Marketplace](https://marketplace.digitalocean.com/apps/teleport). Once your
+  droplet is ready, SSH into the droplet and follow the configuration wizard.
 
-Once you are ready to learn more about Teleport, read our [Core Concepts
-guide](./core-concepts.mdx), which introduces the components of a Teleport
-cluster. You can refer to this glossary as you continue through the
-documentation.
+  </Admonition>
 
-## Choose an edition
+- A two-factor authenticator app such as [Authy](https://authy.com/download/),
+  [Google Authenticator](https://www.google.com/landing/2step/), or [Microsoft
+  Authenticator](https://www.microsoft.com/en-us/account/authenticator)
 
-After trying out Teleport, you are ready to deploy a cluster to your
-infrastructure. Teleport has four editions: 
+You must also have one of the following:
+- A registered domain name.
+- An authoritative DNS nameserver managed by your organization, plus an existing
+  certificate authority. If using this approach, ensure that your browser is
+  configured to use your organization's nameserver.
 
-- Teleport Team
-- Teleport Enterprise Cloud
-- Teleport Enterprise
-- Teleport Community Edition
+### Step 1/4. Configure DNS
 
-You can compare these in our [Choose an
-Edition](./choose-an-edition/introduction.mdx) section.
+Teleport uses TLS to provide secure access to its Proxy Service and Auth
+Service, and this requires a domain name that clients can use to verify
+Teleport's certificate. Set up two DNS `A` records, each pointing to the IP
+address of your Linux host. Assuming `teleport.example.com` is your domain name,
+set up records for:
 
-<Notice type="tip">
+|Domain|Reason|
+|---|---|
+|`teleport.example.com`|Traffic to the Proxy Service from users and services.|
+|`*.teleport.example.com`|Traffic to web applications registered with Teleport. Teleport issues a subdomain of your cluster's domain name to each application.|
 
-You can view information specific to an edition of Teleport by using the "Open
-Source", "Enterprise", and "Cloud" buttons at the top of the page.
+### Step 2/4. Set up Teleport on your Linux host
 
-</Notice>
+#### Install Teleport
+
+On your Linux host, run the following command to install the Teleport binary:
+
+```code
+$ curl https://goteleport.com/static/install.sh | bash -s (=teleport.version=)
+```
+
+#### Configure Teleport
+
+Generate a configuration file for Teleport using the `teleport configure` command.
+This command requires information about a TLS certificate and private key.
+
+(!docs/pages/includes/tls-certificate-setup.mdx!)
```

Comment:
Just a suggestion: For parity, the "Private network deployment" tab in this partial could be updated to include an example of the relevant generated config, as the first tab does.

Reply:
I've suggested removing the sample config from the "Let's Encrypt" tab. That example config is a little incorrect/misleading anyway, since it doesn't show the `cluster_name` field and only includes the Proxy Service config.